xio-core/build.gradle

```diff
@@ -158,3 +140,35 @@ sourceSets {
     }
   }
 }
+
+bintray_package = 'xio'
+publishing {
+  publications {
+    maven(MavenPublication) {
+      groupId project_group
+      artifactId bintray_package
+      version version
+      from components.java
+    }
+  }
+}
+
+bintray {
+  user = bintray_user //this usually comes from gradle.properties file in ~/.gradle
+  key = bintray_apikey //this usually comes from gradle.properties file in ~/.gradle
+  publications = ['maven'] //this refers to publishing.publications.maven up above ^^^
+  dryRun = bintray_dryrun.toBoolean() //Whether to run this as dry-run, without deploying
+  publish = true //If version should be auto published after an upload
+  pkg {
+    userOrg = bintray_subject
+    repo = bintray_repo
+    name = bintray_package
+    version {
+      name = project.version //Bintray logical version name
+    }
+    licenses = ['Apache-2.0']
+    websiteUrl = 'https://github.com/xjdr/xio'
+    issueTrackerUrl = 'https://github.com/xjdr/xio/issues'
+    vcsUrl = 'https://github.com/xjdr/xio.git'
+  }
+}
```

Comment:
I tried to get rid of all of this boilerplate by using `maven-publish` like so: https://github.com/ajoberstar/gradle-bintray#archived

But the ;publish=1 bintray hack isn't supported by gradle right now.